app/assets/stylesheets/layout/common/_contact_panels.scss

```diff
@@ -1,8 +1,18 @@
+.l-contact-panels {
+  margin-top: 40px;
+  border-top: $default-border-thin;
+}
+
 .l-contact-panels--no-border-top {
   margin-top:40px;
   border-top:none;
```

Comment:
You could use `0` here, rather than, `none` to keep consistent with https://github.com/moneyadviceservice/frontend/pull/912/files#diff-5c242d6e74b724d02a75ab34cc91f369R12